lib/src/robot/client.dart

```diff
@@ -5,17 +5,19 @@ import 'package:grpc/grpc_connection_interface.dart';
 import 'package:logger/logger.dart';
 
 import '../gen/common/v1/common.pb.dart';
-import '../gen/robot/v1/robot.pbgrpc.dart';
+import '../gen/google/protobuf/struct.pb.dart';
+import '../gen/robot/v1/robot.pbgrpc.dart' as rpb;
```

Comment:
import aliasing robot pb to avoid identifier collision between the new `DiscoveryQuery` in the SDK, and the pb `DiscoveryQuery`